client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +229,291 @@ def process(self, instance):
     def get_members_and_roots(self, instance):
         return instance[:], instance.data.get("setMembers")
 
+    @classmethod
+    def get_attribute_defs(cls):
+        if not cls.overrides:
+            return []
+
+        override_defs = {
+            "attr": TextDef(
+                "attr",
+                label="Custom Attributes",
+                placeholder="attr1;attr2",
```

Comment:
Match placeholder text of `attrPrefix`
```suggestion
                placeholder="attr1; attr2; ...",
```